tests/snapshots/assets__check_file@packages-tidy-new-parser.typ-40.snap

```diff
@@ -338,14 +354,19 @@ snapshot_kind: text
           name: curry.captures.first(),
           rest: state
             .unfinished-param
-            .slice(processed-chars + curry.end),
+            .slice(processed-chars
+              + curry.end),
         )
       }
     }
-    if args.len() > 0 and (
-      state
-        .unfinished-param
-        .ends-with(",") or state.state == "finished"
+    if (
+      args.len() > 0
+        and (
+          state
+            .unfinished-param
+            .ends-with(",")
+            or state.state == "finished"
+        )
     ) {
```

Comment:
the original snapshot is interesting. Maybe the paren here is not needed

Reply:
the paren here is necessary for associativity